MobiFlight/MobiFlightCache.cs

```diff
@@ -42,23 +42,21 @@ public class MobiFlightCache : MobiFlightCacheInterface, ModuleCacheInterface
         public event EventHandler LookupFinished;
 
 
+#pragma warning disable IDE0044 // Add readonly modifier.
         private volatile List<MobiFlightModuleInfo> AvailableComModules = new List<MobiFlightModuleInfo>();

```

Comment:
VS really wants this to be marked as readonly, but you can't do readonly and volatile at the same time, so ignore the warning.